sdk/core/azure-core/test/libcurl-stress-test/Dockerfile

```diff
@@ -3,11 +3,21 @@
 
 FROM mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
 
-# copy the tagrget binary
-COPY ./bin/libcurl-stress-test/azure-core-libcurl-stress-test ./azure-core-libcurl-stress-test
-RUN chmod +x ./azure-core-libcurl-stress-test
-
+RUN su
```

Comment:
Containers run as root by default so this shouldn't be necessary.

Reply:
done